README.md

```diff
@@ -9,4 +9,42 @@ Online examples: https://blazor.syncfusion.com/demos/file-upload/default-functio
 Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) and [.NET Core SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) or later version in your machine before starting to work on this project.
 
 ## How to run this application
-To run this application, you need to first clone the `getting-started-with-blazor-file-upload-component` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
\ No newline at end of file
+To run this application, you need to first clone the `getting-started-with-blazor-file-upload-component` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+### Validation
+Users can add validation to check file size limits with minimum and maximum ranges, number of files required, total size, and MIME type before uploading a file to a server.
+
+### Drag and drop
+The Blazor File Upload component can upload multiple files by simply dragging them from the file explorer to the drop area (drop zone), which is a more user-friendly way to select and upload multiple files.
+
+### Upload large files
+Manage large file uploads efficiently without any performance degradation using chunk upload. The large files are sliced into small chunks and asynchronously uploaded to the server in a sequential order.
+
+
+## Not sure how to create your first Blazor File Upload? Our documentation can help.
+* [Getting Started with Blazor Server-Side File Upload](https://blazor.syncfusion.com/documentation/file-upload/getting-started?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-file-upload-github-examples)

```

Comment:
check and change utm in all paces
<img width="797" alt="image" src="https://github.com/SyncfusionExamples/getting-started-with-blazor-file-upload-component/assets/61148953/39be2631-0218-4cea-bd25-c6acd00eda03">

Reply:
Changed